### PR TITLE
New tracking

### PR DESCRIPTION
**Pull request by Magnus12**
1. Adding new tracking API docs and remove old.
2. Removed the pooping bear from the FAQ
3. Added some extra info about the VAT amount.

### CODE REVIEW

File: Product Import/tracking/index.rst [hunk 1]
```diff
@@ -1,20 +1,98 @@
...
+
+Overview
+----
+::
+
+    https://mis.cdon.com/deliveries
+
+
+Get the status of your last 100 deliveries. (Latest first)
+The response will be a json file with a list of the deliveries::
```
Comment (wlous) [inline Product Import/tracking/index.rst]: I think you usually write JSON with capital letters

File: Product Import/tracking/index.rst [hunk 2]
```diff
@@ -1,20 +1,98 @@
...
+            "status": "Pending",
+            "errorMessage": null,
+            "totalProducts": 1,
+            "totalPending": 1,
+            "totalSucceeded": 0,
+            "totalFailed": 0
+        }
+    ]
+    
+It is also posible to specify how many records you want to return by adding the querystring "take"::
```
Comment (wlous) [inline Product Import/tracking/index.rst]: possible with two "s"

File: Product Import/tracking/index.rst [hunk 3]
```diff
@@ -1,20 +1,98 @@
...
+The maximum supported value is 1000.
+
+Delivery status
+----
+::
+
+    https://mis.cdon.com/deliveries/<ReceiptId>
 
-The base URL to the tracking-API is::
+Get the status of a aspecific delivery using the ReceiptId.
```
Comment (wlous) [inline Product Import/tracking/index.rst]: Add a space betweem "a" and "specific"

File: Product Import/tracking/index.rst [hunk 4]
```diff
@@ -1,20 +1,98 @@
...
+    
+Product failures
+----
+::
 
-	https://mis.cdon.com/import/<query>
+    https://mis.cdon.com/deliveries/<ReceiptId>/failures
 
-where `<query>` is :doc:`status <importstatus>`, :doc:`summary <importsummary>` or :doc:`details <productdetails>`.
+If a delivery has one of more failed products, details of these failures can be viewed using this endpoint.
```
Comment (wlous) [inline Product Import/tracking/index.rst]: I think "of " should be "or" in this sentence?

File: Product Import/tracking/index.rst [hunk 5]
```diff
@@ -1,20 +1,98 @@
...
+The response will be a json file with a list of the failed products of the specified delivery::
```
Comment (wlous) [inline Product Import/tracking/index.rst]: JSON in capital letters?

File: Product Import/api/price.rst [hunk 6]
```diff
@@ -48,6 +48,7 @@ The ``Vat`` allows different values depending on VAT rates in the market where t
  	12
 
 which represents a VAT of 12%.
+The VAT amount must be valid for the VAT identificatiohn number used in desired market.
```
Comment (wlous) [inline Product Import/api/price.rst]: - "identification" without the "h"
- add "the" before "desired"

**Comment by Magnus12** [comment]
Tack @wlous :-)

**Pull request merged**